templates/artist/includes/artist_detail_updates.html

```diff
@@ -31,12 +19,6 @@
                                 {{ form.image.label }}:
                                 {{ form.image }}
                             </p>
-                            <h5>-- OR --</h5>
```

Comment:
Do you want to remove the feature that gives the ability to upload an image URL? If so, I should remove this from the backend too.